### PR TITLE
add semantic markup and body classes

### PR DESCRIPTION
**Pull request by stefenphelps**

### CODE REVIEW

**Comment by gcorne** [comment]
Nice! Do you mind adjusting the formatting of the code so that the use of line breaks and spaces matches the original?

**Comment by stefenphelps** [comment]
Sure thing. Looks like my jsprettier on save tried and failed at prettifying the HubL markup 🙃

**Comment by stefenphelps** [comment]
@gcorne formatting is fixed now 🙌

File: src/templates/partials/footer.html [hunk 1]
```diff
@@ -2,13 +2,14 @@
   templateType: global_partial
   label: Page footer
 -->
-<div class="footer">
+<footer class="footer">
   <div class="footer__container">
-    {% module "footer_social" path="../../modules/Social follow" label="Social follow" %}
+    {% module "footer_social" path="../../modules/Social follow" label="Social
+    follow" %}
```
Comment (gcorne) [inline src/templates/partials/footer.html]: Extra line break here

File: src/templates/partials/footer.html [hunk 2]
```diff
@@ -2,13 +2,14 @@
...
     <div class="footer__copyright">
-      {% module_block module "footer_copyright_text" path="@hubspot/text" %}
-        {% module_attribute "value" %}© {{ year }} All rights reserved.{% end_module_attribute %}
-      {% end_module_block %}
+      {% module_block module "footer_copyright_text" path="@hubspot/text" %} {%
+      module_attribute "value" %}© {{ year }} All rights reserved.{%
+      end_module_attribute %} {% end_module_block %}
```
Comment (gcorne) [inline src/templates/partials/footer.html]: Formatting here needs to be fixed

File: src/templates/partials/header.html [hunk 3]
```diff
@@ -11,7 +11,8 @@
       {% module "site_search" path="@hubspot/search_input", label="Search" %}
     </div>
     <div class="header__navigation">
-      {% module "navigation-primary" path="../../modules/Menu section", label="Primary Navigation section" %}
+      {% module "navigation-primary" path="../../modules/Menu section",
+      label="Primary Navigation section" %}
```
Comment (gcorne) [inline src/templates/partials/header.html]: Formatting here is a bit off

**Comment by gcorne** [comment]
BTW, I will also work on adding a [.prettierignore](https://prettier.io/docs/en/ignore.html) so that hopefully the html+hubl templates don't get auto-formatted.

**Comment by gcorne** [comment]
Thanks @stefenphelps !

**Pull request merged**